dask/dataframe/tests/test_arithmetics_reduction.py

```diff
@@ -688,11 +689,51 @@ def test_reductions(split_every):
     # nunique is performed using drop-duplicates
     assert_dask_graph(ddf1.b.nunique(split_every=split_every), 'drop-duplicates')
 
+    # testing index
     assert_eq(ddf1.index.min(split_every=split_every), pdf1.index.min())
     assert_eq(ddf1.index.max(split_every=split_every), pdf1.index.max())
     assert_eq(ddf1.index.count(split_every=split_every), pd.notnull(pdf1.index).sum())
 
 
+@pytest.mark.parametrize('frame,axis,out',
+                         [(pd.DataFrame({'a': [1, 2, 3],
+                                         'b': [4, 5, 6]},
+                                        index=[0, 1, 3]),
+                          0, pd.Series([])),
+                          (pd.DataFrame({'a': [1, 2, 3],
+                                        'b': [4, 5, 6]},
+                                        index=[0, 1, 3]),
+                           1, pd.Series([])),
+                          (pd.Series([1, 2.5, 6]), None, None)])
+@pytest.mark.parametrize('redfunc', ['sum', 'prod', 'min', 'max', 'mean', 'var', 'std'])
+def test_reductions_out(frame, axis, out, redfunc):
+    dsk_in = dd.from_pandas(frame, 3)
+    dsk_out = dd.core.Scalar({('a', 0): np.int64(0)}, 'a', 'i8')
```

Comment:
I recommend avoiding creating Dask objects by hand.  This API is less reliable than the user/pandas API.  Instead I recommend creating a scalar using Pandas API on a dask dataframe:

    dsk_in = dd.from_pandas(frame, 3)
    dsk_out = dsk_in.a.sum()